docs/docs/documentation/getting-started/installing-zelthy/manual.mdx

```diff
@@ -196,12 +210,22 @@ docker run --name zango_redis -d -p 6379:6379 redis
 
 ### Starting the celery worker
 
+For linux or MacOs:
+
 Perform the below steps from the root directory of the project
 
 ```
 celery -A <project_name> worker -l INFO
 ```
+For Windows:
+
+Perform the below steps from the root directory of the project
 
+```
+pip install gevent
+celery -A <module> worker -l info -P eventlet
+```
+If any issues comes for eventlet , install event let as well
```

Comment:
this should be gevent and eventlet right?

Reply:
changed it